frontstage_api/controllers/collection_instrument_controller.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from structlog import wrap_logger
+
+from frontstage_api import app
+from frontstage_api.common.request_handler import request_handler
+from frontstage_api.exceptions.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_collection_instrument_size(collection_instrument_id):
+    logger.debug('Retrieving collection instrument size', collection_instrument_id=collection_instrument_id)
+    url = app.config['RAS_CI_SIZE'].format(collection_instrument_id)
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'], fail=False)
+
+    if response.status_code != 200:
+        logger.error('Failed to retrieve collection instrument size', collection_instrument_id=collection_instrument_id)
+        return 0
+    collection_instrument_size = response.json().get('size')
+
+    logger.debug('Successfully retrieved collection instrument size', collection_instrument_id=collection_instrument_id)
+    return collection_instrument_size
+
+
+def download_collection_instrument(collection_instrument_id):
+    logger.debug('Downloading collection instrument', collection_instrument_id=collection_instrument_id)
+    url = app.config['RAS_CI_DOWNLOAD'].format(collection_instrument_id)
+    response = request_handler('GET', url, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        logger.error('Failed to download collection instrument', collection_instrument_id=collection_instrument_id)
+        raise ApiError(url, response.status_code)
+
+    logger.debug('Successfully downloaded collection instrument', collection_instrument_id=collection_instrument_id)
+    return response.content, response.headers.items()
+
+
+def upload_collection_instrument(upload_file, case_id):
+    logger.info('Attempting to upload collection instrument', case_id=case_id)
```

Comment:
Is there a reason this log statement is info but for most other functions it is set to debug?

Reply:
Will change to debug